config/routes.rb

```diff
@@ -1,4 +1,19 @@
 Rails.application.routes.draw do
+
+  root 'welcome#index'
+  post '/books/:id/upvote' => 'books#upvote', as: :upvote_books
+  post '/movies/:id/upvote' => 'movies#upvote', as: :upvote_movies
+  post '/albums/:id/upvote' => 'albums#upvote', as: :upvote_albums
+
+  resources :movies do
+  end
```

Comment:
If you're not including any extra routes, you don't need the extra `do` and `end`. Instead, you can just write `resources :movies`.